Sources/SwiftProtobuf/Enum.swift

```diff
@@ -16,19 +16,26 @@
 // -----------------------------------------------------------------------------
 
 /// Generated enum types conform to this protocol, which provides the
-/// hashability requirement for enums as well as the name mapping requirement
```

Comment:
I would drop the text about the "hashability requirement" as well, since that's obvious from the declaration conforming to `Hashable`.